xylose/scielodocument.py

```diff
@@ -685,6 +685,30 @@ def update_date(self):
 
         return tools.get_date(self.data['v941'][0]['_'])
 
+    @property
+    def mission(self):
+        """
+        This method retrieves the mission of journal.
+        This method deals with the legacy fields (901).
+
+        Return a dict like this:
+            {
+            "es": "La missión es..",
+            "pt": "A missão é.. ",
+            "en": "The mission is"
+            }
+
+        """
+        if 'v901' not in self.data:
+            return None
+
+        missions = {}
+        for mission in self.data.get('v901', []):
+            if 'l' in mission and '_' in mission:
```

Comment:
Se essa condição for inválida para todos as ocorrências de missão será retornado um {} enquanto que de acordo com o Xylose, todos os metodos devem retornar None na ausência de dados.